src/main/java/seedu/address/commons/util/AppUtil.java

```diff
@@ -23,7 +23,7 @@ public static Image getImage(String imagePath) {
      *
      * @throws IllegalArgumentException if {@code condition} is false.
      */
-    public static void checkArgument(Boolean condition) {
+    public static void validateArgument(Boolean condition) {
         if (!condition) {
             throw new IllegalArgumentException();
         }
```

Comment:
Any reason why checkArgument was changed to validateArgument, when both methods had the same logic?

Reply:
Yeah i wanted to use checkArgument to do a weaker check, but now i think it's not necessary.